pkg/monitor/monitor.go

```diff
@@ -55,44 +63,52 @@ func Monitor(clusterName, clusterDomain, templatePath, cfgPath, apiVip string, a
 			if err != nil {
 				return err
 			}
-			newConfig = &config
-			if oldConfig == nil || !cmp.Equal(*oldConfig, *newConfig) {
-				log.WithFields(logrus.Fields{
-					"newConfig": *newConfig,
-				}).Info("Config change detected")
-				err = render.RenderFile(cfgPath, templatePath, RuntimeConfig{LBConfig: newConfig})
-				if err != nil {
-					log.WithFields(logrus.Fields{
-						"config": *newConfig,
-					}).Error("Failed to render HAProxy configuration")
-					return err
-				}
-				_, err = conn.Write([]byte("reload\n"))
-				if err != nil {
-					log.WithFields(logrus.Fields{
-						"socket": haproxyMasterSock,
-					}).Error("Failed to write reload to HAProxy master socket")
-					return err
-				}
+			curConfig = &config
+			if appliedConfig == nil || !cmp.Equal(*appliedConfig, *curConfig) {
+			    if prevConfig == nil || cmp.Equal(*prevConfig, *curConfig) {configChangeCtr++}else {configChangeCtr=1}
```

Comment:
This may be my python background speaking, but I'm not a big fan of putting if...else statements on one line.

And in fact, neither is go fmt. I'm not sure why the ci job didn't catch this, but if I run go fmt against this change locally I get a large number of changes. That should probably be cleaned up before it merges.